lib/tasks/publishing_api.rake

```diff
@@ -188,6 +188,26 @@ namespace :publishing_api do
     puts "Finished queuing items for Publishing API"
   end
 
+  desc "Republish all editions with official attachments"
+  task republish_editions_with_official_attachments: :environment do
+    editions = Edition.where(
+      id: (
+        (
+          Attachment.where.not(unnumbered_hoc_paper: [nil, false]).pluck(:attachable_id) + Attachment.where.not(unnumbered_command_paper: [nil, false]).pluck(:attachable_id) + Attachment.where.not(command_paper_number: nil).pluck(:attachable_id) + Attachment.where.not(hoc_paper_number: nil).pluck(:attachable_id)
```

Comment:
This line is rather long, could it be formatted differently?

Reply:
yeah, it also didn't do what I wanted it to. This has been shortened some and fixed, it's still quite long though.